encodings/fastlanes/src/rle/compress.rs

```diff
@@ -144,7 +168,7 @@ where
         range.finish();
     }
 
-    let offset_within_chunk = array.offset_in_chunk(array.offset);
+    let offset_within_chunk = array.offset();
     let primitive_array = builder.finish_into_primitive();
```

Comment:
If all you're doing with this builder is just set values and not validity you can simplify the logic by creating a BufferMut<T>, the builder lets you associate validity as well but in this case we need to mutate the built array to slice values at the end